daffodil-test/src/test/resources/org/apache/daffodil/section07/variables/variables.tdml

```diff
@@ -2646,6 +2649,9 @@
         </root>
       </tdml:dfdlInfoset>
     </tdml:infoset>
+    <tdml:warnings>
+      <tdml:warning>State was: VariableRead</tdml:warning>
+    </tdml:warnings>
```

Comment:
I think this `<tdml:warnings>` can be removed. Although this test might result in that warning, it really isn't specifically about that warning--it's really just making sure some escape character thing works. Whether or not there is a warning doesn't really matter to this tests.

The above tests that you've modified is specifically about this warning and making sure it's reasonable and correct.